cf/__init__.py

```diff
@@ -81,8 +81,8 @@
 """
 
 __Conventions__ = "CF-1.8"
-__date__ = "2021-05-25"
-__version__ = "3.9.0"
+__date__ = "2021-??-??"
+__version__ = "3.10.0+regrid"
```

Comment:
I had to remove the `+regrid` to `pip install - e.` to test, so assuming you will cut that off, all is good!